specification/gedcom-03-datamodel.md

```diff
@@ -1507,11 +1507,13 @@ See also `INDIVIDUAL_EVENT_STRUCTURE`.
 An [Individual Event](#individual-events).
 See also `INDIVIDUAL_EVENT_STRUCTURE`.
 
-#### `BURI` (Burial) `g7:BURI`
+#### `BURI` (Disposal of remains) `g7:BURI`
 
 An [Individual Event](#individual-events).
 See also `INDIVIDUAL_EVENT_STRUCTURE`.
 
+Although defined as any disposal of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "disposal of remains by burial" by some applications and users.
+In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of disposal of remains.
```

Comment:
```suggestion
Although defined as any depositing of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "depositing of remains by burial" by some applications and users.
In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of depositing remains.

```